spec/runners/unified/crud_operations.rb

```diff
@@ -244,6 +244,9 @@ def aggregate(op)
       if batch_size = args.use('batchSize')
         opts[:batch_size] = batch_size
       end
+      if !(allow_disk_use = args.use('allowDiskUse')).nil?
```

Comment:
When this option was false, `allow_disk_use = args.use('allowDiskUse')` would evaluate to false and the code inside it would not execute. I had to get a little creative here.

Reply:
Another way that hopefully works with whatever class `args` is should be `if args.key?('allowDiskUse')`.